Kernel/Bus/USB/USBHub.cpp

```diff
@@ -72,7 +72,7 @@ ErrorOr<void> Hub::enumerate_and_power_on_hub()
     if constexpr (USB_DEBUG) {
         dbgln("USB Hub Descriptor for {:04x}:{:04x}", m_vendor_id, m_product_id);
         dbgln("Number of Downstream Ports: {}", descriptor.number_of_downstream_ports);
-        dbgln("Hub Characteristics: 0x{:04x}", descriptor.hub_characteristics);
+        dbgln("Hub Characteristics: 0x{:04x}", static_cast<u16>(descriptor.hub_characteristics));
```

Comment:
Is this the correct solution?
Maybe we need to use `ByteReader::load<u16>(...)` here instead?

Reply:
Well this seemed to work for me and is afaik the intended way of making am explicit copy